.circleci/config.yml

```diff
@@ -434,4 +434,4 @@ jobs:
           name: Delete cluster
           command: |
             make cluster-down
-          no_output_timeout: 30m
+          no_output_timeout: 30m
```

Comment:
and this?

Reply:
Came, as part of the branch merge. No manual changed involved in this PR.